bsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tablecloth",
```

Comment:
Why the name change here?

Reply:
This, and the other strange changes were the result of some careless rebasing, sorry.

Reply:
That said, how come you went for `tablecloth-bucklescript` and `tablecloth-native`?

I was going to bring this up aaaages ago but kinda forgot about it (then the whole Standard thing happened) and actually grabbed the `tablecloth` npm package just in case.